src/esm_runscripts/namelists.py

```diff
@@ -90,6 +90,71 @@ def nmls_load(mconfig):
                 mconfig["namelists"][nml].uppercase = True
         return mconfig
 
+    @staticmethod
+    def nmls_check_changes(namelist_changes):
+        """
+        Checks if namelist changes are given in correct syntax.
+        If not, a user_error will be raised and stopp the execution.
+        
+        Programmer Information
+        ----------------------
+        
+        Parameters
+        ----------
+        namelist_changes : nested dict
+        
+        If the syntax is correct, namelist_changes should be a nested dict of the following form:
+        {'namelist1.nml': {'namelist1': {'variable1': 'value1', 'variable2': 'value2', 'variable3': 'value3'}, 'namelist2': {'variable1': value1}}}
```

Comment:
```suggestion
        If the syntax is correct, namelist_changes should be a nested dict of the following form::

        {'namelist1.nml': {'namelist1': {'variable1': 'value1', 'variable2': 'value2', 'variable3': 'value3'}, 'namelist2': {'variable1': value1}}}
```
This should make sphinx generate a code box for the namelist dictionary, when building the documentation